Artesian/Artesian.SDK/Service/Clients/Client.cs

```diff
@@ -40,11 +40,11 @@ internal sealed class Client
         private readonly LZ4MessagePackContentSerializer _lz4msgPackSerializer;
 
         private readonly string _url;
-        private readonly AsyncPolicy _resilienceStrategy;
-        private readonly string _apiKey;
+        private readonly AsyncPolicy? _resilienceStrategy;
```

Comment:
resilienceStrategy must be not null. change the GerResilienceStrategy() accordling